src/main/kotlin/no/nb/mlt/wls/domain/model/ItemCategory.kt

```diff
@@ -0,0 +1,11 @@
+package no.nb.mlt.wls.domain.model
+
+enum class ItemCategory {
+    PAPER,
+    DISC,
+    FILM,
+    PHOTO,
+    EQUIPMENT,
```

Comment:
Need to consider how we want to name this category, "gjenstand" translates pretty much into object, but that's not fitting. Equipment is a possibility; other names we could give it:

- hardware
- appliance
- resource
- gadget
- apparatus
- paraphernalia
- accoutrement
- appurtenance
- "bits and bobs" (could also rename bulk items to "odds and sods")

Reply:
As discussed this remains EQUIPMENT for now. Odds and sods unfortunately did not make this years pageant, but maybe next year